dev-haskell/cgi/cgi-3001.5.0.1.ebuild

```
@@ -1,33 +1,32 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-# ebuild generated by hackport 0.6.1.9999
+# ebuild generated by hackport 0.9.0.0.9999
 #hackport: flags: -old-mtl
```

This will make sure hackport automatically changes this flag name in the future (assuming it didn't already?)

```suggestion
#hackport: flags: -old-mtl,install-examples:examples
```

---

Thanks!